dApp/user/src/lib/Old/Eth.res

```diff
@@ -1,67 +1,47 @@
-type t = BN.t
+open Ethers
+open Utils
+
+type t =BigNumber.t
 
 // Float
 let toFixedWithPrecisionNoTrailingZeros = (number: float, ~digits) =>
   number->Js.Float.toFixedWithPrecision(~digits)->float_of_string->Float.toString
 
-@deriving(jsConverter)
-type ethUnit = [
-  | #wei
-  | #kwei
-  | #mwei
-  | #gwei
-  | #microether
-  | #milliether
-  | #ether
-  | #kether
-  | #mether
-  | #geher
-  | #tether
-]
-
-@module("web3-utils") external fromWei: (t, string) => string = "fromWei"
-@dead("+toWei") @module("web3-utils") external toWei: (string, string) => string = "toWei"
-
-let fromWeiEth: t => string = value => fromWei(value, ethUnitToJs(#ether))
-
 type getUnit =
   | Eth(ethUnit)
   | Usd(float, int)
 
 let getFloat = (value, unit) =>
   switch unit {
-  | Eth(unit) => fromWei(value, unit->ethUnitToJs)->Js.Float.fromString
-  | Usd(conversion, _) => fromWei(value, #ether->ethUnitToJs)->Js.Float.fromString *. conversion
+  | Eth(unit) =>formatUnits(.value, unit)->Js.Float.fromString
+  | Usd(conversion, _) =>formatEther(value)->Js.Float.fromString *. conversion
   }
 let get = (value, unit) =>
   switch unit {
-  | Eth(unit) => fromWei(value, unit->ethUnitToJs)
+  | Eth(unit) =>formatUnits(.value, unit)
   | Usd(conversion, digits) =>
-    (fromWei(value, #ether->ethUnitToJs)->Js.Float.fromString *. conversion)
+    (formatEther(value)->Js.Float.fromString *. conversion)
       ->toFixedWithPrecisionNoTrailingZeros(~digits)
   }
 
 let make: string => option<t> = wei => {
-  let result = Helper.isPositiveStringInteger(wei) ? Some(BN.new_(wei)) : None
+  let result = Helper.isPositiveStringInteger(wei) ? Some(BigNumber.fromUnsafe(wei)) : None // should be safe if helper fn returns true
   result
 }
 @dead("+makeWithDefault")
 let makeWithDefault: (string, int) => t = (tokenId, default) =>
   switch make(tokenId) {
   | Some(wei) => wei
-  | None => default->Int.toString->BN.new_
+  | None => default ->BigNumber.fromInt
   }
-@dead("+makeFromInt") let makeFromInt: int => t = tokenId => tokenId->Int.toString->BN.new_
+@dead("+makeFromInt") let makeFromInt: int => t = BigNumber.fromInt
```

Comment:
not certain what the @dead annotation means... makeFromInt and makeFromEthStr have analogues which are built into Ethers so maybe we should remove them at some stage?

Reply:
It does nothing, it's from here: https://github.com/rescript-association/reanalyze#dce-controlling-reports-with-annotations (dead code analysis)

It doesn't affect anything or change the behaviour of the code. It is just interesting to run that tool every now and again and see which code isn't being used

Reply:
Ah, thanks.

Reply:
We can actually remove `Eth.res` too, sorry should have said that in the issue. (we should be able to do everything from this file with `Ethers.res`...)

Reply:
Done